mrp_subcontracting_purchase_order/__manifest__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2020 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Link Purchase Order to Subcontract Productions",
+    "version": "12.0.1.0.0",
+    "category": "Manufacturing",
+    "license": "AGPL-3",
```

Comment:
```suggestion
    "license": "LGPL-3",
```
@TimLai125 Please change the license to LGPL.